.github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml

```diff
@@ -107,17 +108,18 @@ jobs:
           echo "py_ver_clean=$PY_VER_CLEAN" >> $GITHUB_OUTPUT
       - name: run Python validatesContainerARM script
         id: run_script
-        run: |
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
           gradle-command: :sdks:python:test-suites:dataflow:py${{steps.set_py_ver_clean.outputs.py_ver_clean}}:validatesContainerARM 
           arguments: |
             -PpythonVersion=${{ matrix.python_version }} \
             -Pcontainer-architecture-list=arm64,amd64 \
             -Pdocker-repository=us.gcr.io/apache-beam-testing/github-actions \
             -Pdocker-tag=${{ env.MULTIARCH_TAG }} \
-            -Ppush-containers
-          env:
-            MULTIARCH_TAG: ${{ env.MULTIARCH_TAG }}
-            USER: github-actions
+            -Ppush-containers \
```

Comment:
is the last line-joining intentional?

Reply:
I just followed what was done here: https://github.com/apache/beam/blob/afa4371b1ddaf017c817d9a833d3a6d04310236c/.github/workflows/beam_PreCommit_Python_Integration.yml#L116